pkg/route/validation/validation.go

```diff
@@ -90,6 +102,33 @@ func validateRoute(route *routev1.Route, checkHostname bool) field.ErrorList {
 		result = append(result, err)
 	}
 
+	if len(route.Spec.HTTPHeaders.Actions.Response) != 0 || len(route.Spec.HTTPHeaders.Actions.Request) != 0 {
+		if route.Spec.TLS != nil && route.Spec.TLS.Termination == routev1.TLSTerminationPassthrough {
+			specTLS := field.NewPath(specTLSTerminationString)
+			result = append(result, field.NotSupported(specTLS, route.Spec.TLS.Termination, []string{"Only edge and re-encrypt routes are supported for providing customized headers."}))
+		}
+	}
+	if len(route.Spec.HTTPHeaders.Actions.Response) != 0 && len(route.Spec.HTTPHeaders.Actions.Response) > 64 {
```

Comment:
```suggestion
	if len(route.Spec.HTTPHeaders.Actions.Response) > 64 {
```
If the len > 64 it is automatically != 0.

Reply:
I wanted to check for nil may be add a nil condition.